modules/nw-install-contour-operator.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * networking/about-contour-operator.adoc
+
+[id="nw-install-contour-operator_{context}"]
+= Installing the Contour Operator
+
+Install the Contour Operator on {product-title} to use link:https://gateway-api.sigs.k8s.io/[Gateway API].
+
+.Prerequisites
+
+* You installed an {product-title} cluster and the `kubectl` command line.
+//* Log in as a user with cluster-admin permission.
+
+.Procedure
+
+. Install Contour Operator:
++
+[source,terminal]
+----
+$ kubectl apply -f https://raw.githubusercontent.com/projectcontour/contour-operator/v1.13.0/examples/operator/operator.yaml
+----
+
+. Verify the availability of the operator:
++
+[source,terminal]
+----
+$ kubectl get deployment/contour-operator -n contour-operator
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME               READY   UP-TO-DATE   AVAILABLE   AGE
+contour-operator   1/1     1            1           12m
+----
+
+. Add `contour` and `contour-certgen` service accounts to the nonroot scc:
```

Comment:
__Note:__ The example uses `projectcontour` (default) as the namespace of the contour/contour-certgen service accounts. Replace `projectcontour` with the namespace used for Contour if you deviate from the example.